apidoc/Titanium/UI/iOS/LivePhotoView.yml

```diff
@@ -14,8 +14,19 @@ extends: Titanium.UI.View
 platforms: [iphone,ipad]
 since: "5.2.0"
 osver: {ios:{min: "9.1"}}
+excludes:
+    events: [click,dblclick,doubletap,longpress,pinch,postlayout,singletap,swipe,
```

Comment:
Didn't find anything. The above example is the same as the Ti.UI.ListView uses for example.